src/composables/useSizes/index.ts

```diff
@@ -15,7 +15,19 @@ export default function useSizes(
 ) {
   const reactiveSize = toValue(windowSize)
     ? useWindowSize()
-    : useElementSize(computed(() => toValue(canvas).parentElement))
+    : useElementSize(
+      computed(
+        () => toValue(canvas).parentElement,

```

Comment:
Hi @ThimoDEV why is this change needed?

Reply:
There is an issue when the TresCanvas uses the parent element width and height.

The usElementSize composable from VueUse loses its reactivity I assume which causes the TresCanvas width and height to be 0. 

This issue occurs when you hit a breakpoint and continue debugging after. When you use the `useWindowSize` prop this doesn't happen.

This fixes that

Reply:
@ThimoDEV would be possible to have this change in a separate PR, is a little bit confusing to understand the scope of this one.